actions/slot_match.py

```diff
@@ -58,3 +58,26 @@ def resolve_slot_match(
                 return resolved_child
 
     return {}
+
+
+def suggest_using_slot_match(
+    user_message: str, slot_match: FuzzySlotMatch, accepted_rate=ACCEPTED_RATIO
+) -> List[str]:
+    user_message = _sanitize_input(user_message)
+    suggestions = {}  # avoiding duplicates
+    for option in slot_match.options:
+        for synonym in option.synonyms:
+            if synonym == "unsure":
```

Comment:
could we use an empty synonyms list  instead?

Reply:
I don't think that'll work if a user types "i dont know what you're talking about" it won't match

probably should make an intent for this and break in the rules? as much as ive been using one

Reply:
As i understand this is one of the options and is hardcoded or comes from an API.
If the synonym is "unsure" it breaks the search of the option ?